packages/example-getting-started/README.md

```diff
@@ -2,6 +2,20 @@
 
 This is the basic tutorial for getting started with Loopback 4!
 
+To get started, jump into the
+[Prerequisites and setup](docs/1-prerequisites-and-setup.md) section.
+
+## Tutorial Steps
+
+1. [Prerequisites and setup](docs/1-prerequisites-and-setup.md)
+1. [Scaffolding your application](docs/2-scaffold-app.md)
+1. [Adding the legacy juggler](docs/3-add-legacy-juggler.md)
+1. [Add your Todo model](docs/4-todo-model.md)
+1. [Add a datasource](docs/5-datasource.md)
+1. [Add a repository](docs/6-repository.md)
+1. [Add a controller](docs/7-controller.md)
+1. [Putting it all together](docs/8-putting-it-together.md)
```

Comment:
Interesting to use `1.` for all items :-). Personally I prefer to number them.

Reply:
Mostly simplifies copy/paste/reordering in the future. Most parsers automatically iterate and number them for you this way.